src/Components/SinglePostCard/SinglePostCard.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+import { BookMarkedIcon, BookmarkOutlineIcon, CommentsIcon, HeartFilledIcon, HeartOutlineIcon } from '../../Assets/AllSVG';
```

Comment:
Nit: can be spread over multi lines for better readability.

Reply:
Prettier automatically changes it back to this.